vllm/executor/ray_gpu_executor.py

```diff
@@ -203,16 +223,6 @@ def _init_workers_ray(self, placement_group: "PlacementGroup",
         self._run_workers("update_environment_variables",
                           all_args=all_args_to_update_environment_variables)
 
-        if len(node_gpus) == 1:
```

Comment:
I believe this is here for a reason. can we keep this?

Reply:
@ruisearch42 any reason to delete this? this is important for single node usecase.

Reply:
@youkaichao I might have ran into issues when having them. But let me try to bring them back.